tests/unit/modules/test_virt.py

```diff
@@ -1392,6 +1394,18 @@ def test_update(self):
         setxml = ET.fromstring(define_mock.call_args[0][0])
         self.assertEqual('vnc', setxml.find('devices/graphics').get('type'))
 
+        # Serial and console test case
+        self.assertEqual({
+                'definition': True,
+                'mem': True,
+                'disk': {'attached': [], 'detached': []},
+                'interface': {'attached': [], 'detached': []}
+            }, virt.update('my vm', serial_type='tcp', console=True, telnet_port=22223))
```

Comment:
with the new parameters concept that would typically be:
`}, virt.update('my vm', serials=[{'type': 'tcp'}], consoles=[{'type': 'tcp', 'port': 22223, 'protocol': 'telnet'}]))`